docs/dotnetcore/introducing-dotnetcore/introducing-dotnetcore.rst

```diff
@@ -0,0 +1,94 @@
+Introducing .NET Core
+=====================
+By `Steve Smith`_ | Originally Published: 1 June 2015 
+
+.. _`Steve Smith`: Author_
+
+.NET Core is a small, optimized runtime that can be targeted by ASP.NET 5 applications. In fact, the new ASP.NET 5 project templates target .NET Core by default, in addition to the full .NET Framework. Learn what targeting .NET Core, or *dnxcore50*, means for your ASP.NET 5 application.
+
+This article covers the following topics:
+	- What is .NET Core?
+	- Motivation Behind .NET Core
+	- .NET Core and ASP.NET
+	- .NET Core and NuGet
+	- Additional Reading
+
+What is .NET Core
+^^^^^^^^^^^^^^^^^
+
+.NET Core 5 is a modular runtime and library implementation that includes a subset of the full .NET Framework. Currently it builds and runs on Windows, but support for Linux and Mac is under development. .NET Core consists of a set of libraries, called "CoreFX", and a small, optimized runtime, called "CoreCLR". .NET Core is open-source, so you can follow progress on the project and contribute to it on GitHub:
+
+	- `.NET Core Libraries (CoreFX) <https://github.com/dotnet/corefx>`_
+	- `.NET Core Common Language Runtime (CoreCLR) <https://github.com/dotnet/coreCLR>`_
+
+The CoreCLR runtime is made available via NuGet (Microsoft.CoreCLR); the CoreFX library is also available as a set of individual NuGet packages, factored according to functionality. These packages are named "System.[module]" on `Nuget.org <http://www.nuget.org/>`_.
+
+One of the key benefits of .NET Core is its portability. You can package and deploy the CoreCLR with your application, eliminating your application's dependency on an installed version of the full .NET Framework at the device or server level. You can host multiple applications side-by-side using different versions of the CoreCLR, and upgrade them individually, rather than being forced to upgrade all of them simultaneously.
```

Comment:
This is a bit of a Windows way of looking at it. The central .NET Framework issue is only specific to Windows. It would be great if you could discuss the benefits in a general way and then potentially end the paragraph with how this is even more important on Windows.